src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -428,6 +561,16 @@ combined image rather than an image from each coil.
 
 #### Timing Parameters
 
+{{ MACROS___make_metadata_table(
+   {
+      "NumberOfVolumesDiscardedByScanner": "RECOMMENDED",
+      "NumberOfVolumesDiscardedByUser": "RECOMMENDED",
+      "DelayTime": "RECOMMENDED",
+      "AcquisitionDuration": 'RECOMMENDED, but REQUIRED for sequences that are described with the `VolumeTiming` field and that do not have the `SliceTiming` field set to allow for accurate calculation of "acquisition time"',
```

Comment:
the linebreaks in the table rendering differ for this field in the spec, with the new version being a bit worse, see old version screenshot:

![image](https://user-images.githubusercontent.com/9084751/124444868-531e1e80-dd7f-11eb-97a9-e6d9ebcd40fb.png)

new version (a bit worse)

![image](https://user-images.githubusercontent.com/9084751/124444915-5dd8b380-dd7f-11eb-898b-081d6ab3a8ce.png)


It's minor, but perhaps you easily see what can be changed to have this work better. If not, no worries.

Reply:
I think other changes are probably just making that column slightly narrower overall. I don't see anything in that string that would explain the change.

Reply:
Alright, thanks for checking 👍